src/values/instruction_value.rs

```diff
@@ -277,6 +278,22 @@ impl<'ctx> InstructionValue<'ctx> {
         }
         Ok(unsafe { LLVMSetAlignment(self.as_value_ref(), alignment) })
     }
+    
+    // SubTypes: Only apply to memory access and alloca instructions
+    /// Sets alignment on a memory access instruction or alloca.
+    #[llvm_versions(11.0..=latest)]
```

Comment:
Why was this function split out?

Reply:
LLVM11 introduced some alignment guards, I thought we should not force <11 to use them, but I will move this to a feature check in the same function

Reply:
So older versions allowed alignment 0 but 11 no longer does?

Reply:
I could not find it in the release notes, so I looked for changes in the code and there has been some work on alignments.
Mainly _most_ of the `setAlignment` methods in [`llvm/IR/Core.cpp`](https://github.com/llvm/llvm-project/blob/release/11.x/llvm/lib/IR/Core.cpp) will now call `Align()` instead of `MaybeAlign()` which fails on 0 values. The only exception is `GlobalValue`. 
I can change this back to remove the check but it means setting alignment to 0 will panic instead of returning the Result<>. 
Since this is on instruction_value and instruction will check for 0, it might be worth keeping the check there.
The changes on LLVM for that are : 
https://reviews.llvm.org/D77454
https://reviews.llvm.org/D79968
https://reviews.llvm.org/D80044

Reply:
Great catch. We should just add a feature check in the same function I think? Unless you feel strongly otherwise

Reply:
We really don't want LLVM to hit an assertion under us because that's bad news

Reply:
I already changed it to a feature check in a previous commit : https://github.com/TheDan64/inkwell/pull/224/files#diff-8e2e5aa572ea6e78c6571db5bae3c54a30f55b202b9c0995d76e145013ea96c1